LiveSplit.SuperMeatBoy.asl

```diff
@@ -29,6 +30,9 @@ startup
 		string description = String.Format("Split before boss {0}", world);
 		settings.Add(name, false, description, "bossSplit");
 	}
+		
+	settings.Add("deathDisp", false, "Death count display");
+	settings.Add("createUI", true, "Create UI if needed");
```

Comment:
Could there perhaps be a way to omit needing createUI? Seems not too user-friendly the way it is now.

Reply:
Yea this setting is a holdover from a previous implementation, I suppose it isn't neccessary. I can just remove it

Reply:
Fixed!